container/sagemaker/serve.py

```diff
@@ -169,30 +183,37 @@ def _find_saved_model_files(self, path):
                     yield os.path.join(path, e.name)
 
     def _setup_gunicorn(self):
-        lib_path_exists = os.path.exists(PYTHON_LIB_PATH)
-        requirements_exists = os.path.exists(REQUIREMENTS_PATH)
-        python_path_content = ['/opt/ml/model/code']
+        python_path_content = []
+        use_python_path = ''
```

Comment:
var name suggests boolean but initial value is a string

Reply:
I will fix this